db/migrate/20150808115436_remove_requirement_from_users_invitation_code.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveRequirementFromUsersInvitationCode < ActiveRecord::Migration
+  def change
+    change_column_null :users, :invitation_code, true, ENV['INVITATION_CODE']
```

Comment:
The fourth arg is the value to fill in `invitation_code` if you were to migrate down since the column would be required again. 
```
# The method accepts an optional fourth argument to replace existing
# +NULL+s with some other value. Use that one when enabling the
# constraint if needed, since otherwise those rows would not be valid.

Assuming the ENV variable is set at migration time, then users would remain valid upon rollback. In the happy path of migrating up, the ENV variable will not be used.